default/scripting/buildings/CONC_CAMP.focs.txt

```diff
@@ -39,12 +42,13 @@ BuildingType
                 ]
             ]
             effects = AddSpecial name = "CONC_CAMP_MASTER_SPECIAL"
+
         EffectsGroup
             scope = Source
-            activation =    ContainedBy And [
-                                Object id = Source.PlanetID
-                                Population high = 0
-                            ]
+            activation = ContainedBy And [
+                    Object id = Source.PlanetID
+                    Population high = 0
+                ]
```

Comment:
weird indenting.  line up the `]` with `a` of `activation` and indent the contained lines by 4 spaces

Reply:
Fixed.